dockerfiles/ci/buster/php-7.0/Dockerfile

```diff
@@ -50,6 +50,8 @@ COPY --chown=circleci:circleci --from=php-debug-zts $PHP_INSTALL_DIR/debug-zts $
 COPY --chown=circleci:circleci --from=php-debug $PHP_INSTALL_DIR/debug $PHP_INSTALL_DIR/debug
 COPY --chown=circleci:circleci --from=php-nts $PHP_INSTALL_DIR/nts $PHP_INSTALL_DIR/nts
 
+USER root
```

Comment:
What's up with this? We're anyway using sudo everywhere in this dockerfile, where we need root?